pages/care-package/[guid]/core-package.js

```diff
@@ -105,7 +123,19 @@ const CorePackage = () => {
     const data = getValues();
     setIsRequestBeingSent(true);
     try {
-      const { id } = await updateCoreCarePackage({ data, packageId });
+      const formData = getFormDataWithFile(data, {
+        file: 'socialWorkerCarePlanFile',
+        fileId: 'socialWorkerCarePlanFileId',
+        fileName: 'socialWorkerCarePlanFileName'
+      });
+
+      if (!data.isS117Client) formData.append('isS117Client', false);
```

Comment:
why haven't you used getFormData ? can make a modified getFormData that returns false instead of null